extensions/notebook/src/jupyter/jupyterServerInstallation.ts

```diff
@@ -312,34 +346,88 @@ export default class JupyterServerInstallation {
 	}
 
 	private async installJupyterProsePackage(): Promise<void> {
+		let installJupyterCommand: string;
 		if (process.platform === constants.winPlatform) {
 			let requirements = path.join(this._pythonPackageDir, 'requirements.txt');
-			let installJupyterCommand = `"${this._pythonExecutable}" -m pip install --no-index -r "${requirements}" --find-links "${this._pythonPackageDir}" --no-warn-script-location`;
+			installJupyterCommand = `"${this._pythonExecutable}" -m pip install --no-index -r "${requirements}" --find-links "${this._pythonPackageDir}" --no-warn-script-location`;
+		}
+
+		if (installJupyterCommand) {
 			this.outputChannel.show(true);
 			this.outputChannel.appendLine(localize('msgInstallStart', "Installing required packages to run Notebooks..."));
-			await utils.executeStreamedCommand(installJupyterCommand, this.outputChannel);
+			await this.executeCommand(installJupyterCommand);
```

Comment:
what's the difference between streamedcommand previously and executeCommand?

Reply:
executeCommand is a wrapper around executeStreamedCommand so that we don't have to keep passing the same arguments again and again.